Assets/Scripts/Managers/SoundManagerScript.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor.UI;
+using UnityEngine;
+
+public class SoundManagerScript : MonoBehaviour
+{
+    private static SoundManagerScript S; // this is the singleton!
```

Comment:
Singleton is usually named Instance.

I wonder if we can store all audio clips with enums and arrays?

Should also add a sfxSource for all global sound effects/UI sound effects.